spec/integration/unhandled_errors_in_receivers_spec.rb

```diff
@@ -5,6 +5,21 @@
 require_relative "support/logging_receiver"
 require_relative "support/integration_test_helpers"
 
+class PoorlyBehavingErrorHandler < Pwwka::ErrorHandlers::BaseErrorHandler
+	def handle_error(receiver,queue_name,payload,delivery_info,exception)
+		puts "whoops, do I break everything behind me?"
+		raise "whoops, do I break everything behind me?"
+    keep_going
+	end
+end
+
+class ErrorHandlerThatWorksFine < Pwwka::ErrorHandlers::BaseErrorHandler
+	def handle_error(receiver,queue_name,payload,delivery_info,exception)
+		puts "we should get here"*80
```

Comment:
unless these puts statements are used in expectation, please remove them to simplify the test.